src/hotspot/share/runtime/continuationWrapper.inline.hpp

```diff
@@ -46,10 +47,8 @@ class ContinuationWrapper : public StackObj {
   oop                _continuation;  // jdk.internal.vm.Continuation instance
   stackChunkOop      _tail;
 
-#if CONT_JFR // Profiling data for the JFR event
   short _e_size;
   short _e_num_interpreted_frames;
```

Comment:
The decalration of these two fields needs to be guarded by `#if CONT_JFR`.

Reply:
I don't see the point of having this mess for two variable initializations.

Reply:
maybe all the JFR code can go in freezethaw as you suggest.

Reply:
I moved the jfr code in continuationFreezeThaw.cpp